janus_core/helpers/mlip_calculators.py

```diff
@@ -217,6 +217,36 @@ def choose_calculator(
         kwargs.setdefault("sevennet_config", None)
         calculator = SevenNetCalculator(model=model, device=device, **kwargs)
 
+    elif arch == "orb":
+        __version__ = "0.3"
+        from orb_models.forcefield.calculator import ORBCalculator
+        from orb_models.forcefield.graph_regressor import GraphRegressor
+        import orb_models.forcefield.pretrained as orb_ff
+
+        if isinstance(model_path, str):
+            match model_path:
```

Comment:
I don't think we can use match case with 3.9?

Reply:
yes it does not but it breaks my heart to write if elif elif elif like we are in fotran 77 world... is not pyhton3.9 eol this month?

Reply:
maybe we shall catch the guilty one and then deal with it separately?

Reply:
I think actually even 3.8 isn't eol until [next month](https://devguide.python.org/versions/#supported-versions), so a little way to go until 3.9.

Reply:
my good intentions destroyed...

Reply:
in fairness they have a dict already we can use that.